CHANGES.rst

```diff
@@ -54,6 +54,7 @@ Pull requests
 - Remove gammapy.shower package [#291] (Christoph Deil)
 - Add cube background model class [#299] (Manuel Paz Arribas)
 - Use assert_quantity_allclose from Astropy [#306] (Manuel Paz Arribas)
+- Change sample_sphere to use Angle objects [#283] (Manuel Paz Arribas)
```

Comment:
Please change the title of this PR and the changelog entry to reflect the content.
Maybe "Consistent random number handling and improve sample_sphere"?